packages/core/src/managed/config/DeliveryConfig.tsx

```diff
@@ -1,24 +1,65 @@
 import React from 'react';
+
 import { RelativeTimestamp } from '../RelativeTimestamp';
+import {
+  FetchApplicationManagementDataDocument,
+  FetchApplicationManagementDataQueryVariables,
+  useImportDeliveryConfigMutation,
+} from '../graphql/graphql-sdk';
+import { useApplicationContextSafe } from '../../presentation/hooks/useApplicationContext.hook';
 import { YamlViewer } from '../utils/YamlViewer';
+import { NotifierService, Spinner } from '../../widgets';
 
 interface IDeliveryConfigProps {
   config?: string;
   updatedAt?: string;
   isProcessed?: boolean;
 }
 
+const ReImportConfig = () => {
+  const appName = useApplicationContextSafe().name;
+  const refetchVariables: FetchApplicationManagementDataQueryVariables = { appName };
+  const [importDeliveryConfig, { error, loading }] = useImportDeliveryConfigMutation({
+    variables: { application: appName },
+    refetchQueries: [{ query: FetchApplicationManagementDataDocument, variables: refetchVariables }],
+  });
+
+  React.useEffect(() => {
+    if (!error) return;
+    NotifierService.publish({
+      key: 'import-error',
+      content: `Failed to import delivery config - ${error.message}`,
+      options: { type: 'error' },
+    });
+  }, [error]);
+
+  return (
+    <>
+      <small>
+        (
```

Comment:
The parenthesis outside of `<button />` are adding spaces around the text, which looks a bit weird IMHO.